README.md

```diff
@@ -237,9 +238,9 @@ run on real browser, either because the tests is exercising browser quirks or be
 the test requries features that jsdom does not provide, putting them into a
 `e2e.spec.js` file is completely valid and good.
 
-TODO(awong): Figure out sauce labs integrations. Not all e2e tests should always be
-run on all browsers. That's wasteful. How do we determine what should be run on
-multiple browsers as opposed to on PhantomJS in Jenkins?
+### End-to-end Test -- Sauce Labs
+
+TODO: Translate internal documentation on running tests with Sauce Labs into something public.
```

Comment:
Should we just delete this section altogether since we can't share our Sauce Labs credentials publicly?

Reply:
This is our internal documentation: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Cross-Platform%20Testing/cross-platform-testing-plan.md

Reply:
:thinking: Yeah, I can see it go either way, but I think it wouldn't be super useful without the credentials.

Reply:
OK, removed the section.